gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/mapper/UserMapper.java

```diff
@@ -220,6 +221,16 @@ public static io.gravitee.am.model.User convert(User scimUser) {
                     throw new UserInvalidException("Unable to parse lastPasswordReset. lastPasswordReset must be in ISO 8601 format.");
                 }
             }
+            var preRegistration = graviteeUser.getAdditionalInformation().get(PRE_REGISTRATION_KEY);
+            if (preRegistration != null) {
+                if (preRegistration instanceof Boolean) {
+                    user.setPreRegistration((Boolean) preRegistration);
+                    user.setPassword(null);//user will receive email to set password.
+                } else {
+                    LOGGER.error("preRegistration must be boolean");
+                    throw new UserInvalidException("Unable to parse preRegistration. lastPasswordReset must be boolean.");
+                }
```

Comment:
suggestion: maybe we should remove this value from the additionalInfo as it has dediacted attribute for it in the user profile. wdyt ?